test/spec/CMakeLists.txt

```diff
@@ -71,9 +80,8 @@ wasmedge_add_library(wasmedgeTestSpec
 )
 
 target_link_libraries(wasmedgeTestSpec
-  PRIVATE
-  RapidJSON
   PUBLIC
+  simdjson
```

Comment:
Do you have any intention of making `simdjson` a PUBLIC scope?

Reply:
No, apologies. Will use SIMDJSON under PRIVATE scope

Reply:
I have added SIMDJSON under PRIVATE scope in [e7162ac](https://github.com/WasmEdge/WasmEdge/pull/2659/commits/e7162acd4ff9812fa00a628c6d35802cb09d6ea6)